pkg/scheduler/actions/preempt/preempt_test.go

```diff
@@ -18,15 +18,185 @@ package preempt
 
 import (
 	"testing"
+	"time"
 
-	"github.com/kubernetes-sigs/volcano/pkg/scheduler/framework"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
 
-	"github.com/kubernetes-sigs/volcano/pkg/scheduler/plugins/drf"
+	kbv1 "github.com/kubernetes-sigs/volcano/pkg/apis/scheduling/v1alpha1"
+	"github.com/kubernetes-sigs/volcano/pkg/scheduler/api"
+	"github.com/kubernetes-sigs/volcano/pkg/scheduler/cache"
+	"github.com/kubernetes-sigs/volcano/pkg/scheduler/conf"
+	"github.com/kubernetes-sigs/volcano/pkg/scheduler/framework"
+	"github.com/kubernetes-sigs/volcano/pkg/scheduler/plugins/conformance"
+	"github.com/kubernetes-sigs/volcano/pkg/scheduler/plugins/gang"
+	"github.com/kubernetes-sigs/volcano/pkg/scheduler/util"
 )
 
 func TestPreempt(t *testing.T) {
-	framework.RegisterPluginBuilder("drf", drf.New)
+	framework.RegisterPluginBuilder("conformance", conformance.New)
+	framework.RegisterPluginBuilder("gang", gang.New)
 	defer framework.CleanupPluginBuilders()
 
-	// TODO (k82cn): Add UT cases here.
+	tests := []struct {
+		name      string
+		podGroups []*kbv1.PodGroup
+		pods      []*v1.Pod
+		nodes     []*v1.Node
+		queues    []*kbv1.Queue
+		expected  int
+	}{
+		{
+			name: "one Job with two Pods on one node",
```

Comment:
Confused about the name,  I am not familiar with the scheduler part, though.

Reply:
Those are just test case names

Reply:
Yes, i know. Just curious why one job and where is the job? Overall just a nit.

Reply:
One podGroup is created in that test case, so all pods in a PodGroup is handled as all tasks under a Job in code.  So here only one PodGroup is created, which is internally(in code) one job in that queue.  That is why termed the test case name as one job.

Reply:
ok